enhancements/scorecard-metadata.md

```diff
@@ -0,0 +1,100 @@
+---
+title: scorecard-metadata
+authors:
+  - "@jemccorm"
+  - "@jlanford"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-04-29
+last-updated: 2020-04-29
+status: provisional
+---
+
+# scorecard-metadata
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Summary
+
+This enhancement is to define a place within the bundle format whereby
+scorecard metadata could be added.  This will allow tests to be co-located
+with the normal operator content found in a bundle.  This enhancement
+suggests naming and paths for the scorecard metadata within the bundle.
+
+## Motivation
+
+The purpose of this enhancement is to define a location for scorecard
+and other test assets to be included into the bundle.  This allows
+test frameworks and tools a known place to search for test assets
+within a bundle and reduces confusion to test developers as to 
+where test assets will reside.
+
+### Goals
+
+This enhancement will be used by the scorecard feature within the operator-sdk
+initially as part of a major redesign of scorecard functionality.  Other
+test frameworks apart from operator-sdk or scorecard could leverage this
+enhancement as well.
+
+### Non-Goals
+
+This enhancement is scoped to define the location within the bundle for
```

Comment:
A thought: generalizing this PR to provide a spec for adding arbitrary items to a bundle image (what are the requirements, labels, etc), and using scorecard test definitions as a living example, would allow us to write more succinct bundle-modifying EPs in the future. This is assuming there isn't already a spec for doing so.